package.json

```diff
@@ -29,21 +29,21 @@
     "create-jest-runner": "^0.6.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.1.0",
-    "@babel/core": "^7.1.0",
-    "@babel/preset-env": "^7.1.0",
+    "@babel/cli": "^7.10.4",
```

Comment:
?

Reply:
Just to make it the highest possible, that's all